pkgs/io_file/lib/src/vm_posix_file_system.dart

```diff
@@ -517,6 +517,19 @@ final class PosixFileSystem extends FileSystem {
     );
   });
 
+  @override
+  void removeFile(String path) => ffi.using((arena) {
+    if (libc.unlinkat(
+          libc.AT_FDCWD,
+          path.toNativeUtf8(allocator: arena).cast(),
+          0,
+        ) ==
```

Comment:
This is somewhat unfortunate formatting. Perhaps either the `libc.unlinkat()` result or the `path.toNativeUtf8()` result could be hoisted into a variable, just to improve readability?

Reply:
Done.